Trappist/src/Promact.Trappist.DomainModel/ApplicationClass/ProgramingLanguageEnum.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Promact.Trappist.DomainModel.ApplicationClass
+{
+    public enum ProgramingLanguageEnum
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this enumeration to remove the 'Enum' suffix. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS2344)